utils/sysfs/sysfs.go

```diff
@@ -242,7 +243,11 @@ func (self *realSysFs) GetSystemUUID() (string, error) {
 			//If running on a KVM guest on Power then UUID is /proc/device-tree/vm,uuid
 			id, err = ioutil.ReadFile(path.Join(ppcDevTree, "vm,uuid"))
 			if err != nil {
-				return "", err
```

Comment:
Updated the 'utils/sysfs/sysfs.go' with modified if-else statements. Please check and let me know if it looks ok.